index.html

```diff
@@ -39,9 +54,8 @@ <h3 class="headthree">I am a Software Developer</h3>
         </div>
```

Comment:
You're almost close to the final result, So please update the sizing and spacing issues in here as well.

![image](https://user-images.githubusercontent.com/34779928/194695266-f73abf3a-f510-4cf8-82f4-c34b57a9215f.png)